core/graphelier-service/api/routes.go

```diff
@@ -58,4 +58,10 @@ var routes = Routes{
 		"/instruments/",
 		hndlrs.CustomHandler{H: hndlrs.FetchInstruments},
 	},
+	Route{
+		"RefreshCache",
+		"GET",
+		"/_refresh/",
```

Comment:
i get the use of the `_`, but this isn't really compliant with REST I believe. Can it just be a more descriptive and meaningful route name ?

Reply:
This is purely a "debug" route, therefore I did not think RESTful concerns applied. I can make it more compliant by changing the verb to a POST but it would make it less user friendly as it would not be possible to call it directly from the browser

Reply:
fair enough